ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressInput.service.ts

```diff
@@ -334,7 +336,7 @@ export class AddressInputService implements OnDestroy {
       state: new FormControl(''),
       city: new FormControl('', Validators.required),
       phone: new FormControl(''),
-      guid: new FormControl('')
+      guid: phoneRequired ? new FormControl('', Validators.required) : new FormControl(''),
```

Comment:
You might have applied the validation to wrong the field.

Reply:
d'oh! But that points out something I don't understand: it seemed fine before even if I didn't push required down to this service.